pygments_solarized/__init__.py

```diff
@@ -1,5 +1,7 @@
-from light import SolarizedStyle
-from dark import SolarizedDarkStyle
-from dark256 import SolarizedDark256Style
+from __future__ import absolute_import
+
+from .light import SolarizedStyle
```

Comment:
No relative imports. Change this back to absolute imports.

Reply:
I can do that easily enough and will get it later today if you really want.  May I ask why?  Is there an issue with using relative imports that I may be unaware of?

On master it is currently using relative imports implicitly.  This change just makes that explicit and does it in a way which is compatible with both python 2 and python 3.  This is why when using Python 3 there is an import error running `pygmentize -L styles` with this module installed.

```
*** Error while highlighting:
ModuleNotFoundError: No module named 'light'
   (file "/Users/justin/.local/share/virtualenvs/pygments-solar-L3O5mZx3/lib/python3.6/site-packages/pygments_solarized/__init__.py", line 1, in <module>)
*** If this is a bug you want to report, please rerun with -v.
(pygments-solar-L3O5mZx3) ➜  pygments-solar python --version
Python 3.6.3
```

vs the python 2 final output
```
* solarized_dark:
    Dark version solarized theme (http://ethanschoonover.com/solarized).
* solarized:
    Light version solarized theme (http://ethanschoonover.com/solarized).
* solarized_dark256:
    Dark 256-color version solarized theme (http://ethanschoonover.com/solarized).
(pygments-solar-L3O5mZx3) ➜  pygments-solar python --version
Python 2.7.13
```